askama/Cargo.toml

```diff
@@ -19,12 +19,12 @@ maintenance = { status = "actively-developed" }
 [features]
 default = ["config", "humansize", "num-traits", "urlencode"]
 config = ["askama_derive/config"]
-humansize = ["askama_derive/humansize", "dep_humansize"]
-markdown = ["askama_derive/markdown", "comrak"]
-num-traits = ["askama_derive/num-traits", "dep_num_traits"]
-serde-json = ["askama_derive/serde-json", "askama_escape/json", "serde", "serde_json"]
-serde-yaml = ["askama_derive/serde-yaml", "serde", "serde_yaml"]
-urlencode = ["askama_derive/urlencode", "percent-encoding"]
+humansize = ["askama_derive/humansize", "dep:humansize"]
+markdown = ["askama_derive/markdown", "dep:comrak"]
+num-traits = ["askama_derive/num-traits", "dep:num-traits"]
+serde-json = ["askama_derive/serde-json", "askama_escape/json", "dep:serde", "dep:serde_json"]
```

Comment:
For `serde_json` I think we should prefer the upstream spelling for the feature, now that we can do this. Maybe keep `serde-json` around as an alias (but change any mentions in documentation to the underscore variant)?